metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -0,0 +1,42 @@
+namespace com.linkedin.dashboard
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.ChartUrn
+import com.linkedin.common.Time
+
+/**
+ * Information about a dashboard
+ */
+record DashboardInfo {
+
+  /**
+   * Title of the dashboard
+   */
+  title: string
+
+  /**
+   * Detailed description about the dashboard
+   */
+  description: string
+
+  /**
+   * Charts in a dashboard
+   */
+  charts: array[ChartUrn] = [ ]
+
+  /**
+   * Captures information about who created/last modified/deleted this dashboard and when
+   */
+  lastModified: ChangeAuditStamps
+
+  /**
+   * Access level for the dashboard
+   */
+  access: optional AccessLevel
```

Comment:
curious: is there a need/usecase to store the PII level information for the dashboard? i.e. if the dashboard contains PII or not?

Reply:
There is no need at this point but I think even if we consider, this would be a *derived* metadata for the dashboard. Therefore, it doesn't make sense to store that in the dashboard. A dashboard will contain PII if any of it's input datasets contain PII. I guess we can do that once we open source compliance stuff for datasets.